src/actions/categoriasActions.js

```diff
@@ -1,13 +1,12 @@
 import authAxios from "../config/authAxios";
 import { STATUS_OK } from "../config/constants";
-import response from "../config/network/response";
 
 export const getCategoryById = async (id) => {
   try {
-    const res = await authAxios.get(`/category/${id}`);
+    const res = await authAxios.get(`/artworks/?department=${id}`);
 
-    if (res.data.status === STATUS_OK) {
-      return res.data.body;
+    if (res.status === STATUS_OK) {
+      return res.data.data;
     } else {
       return null;
     }
```

Comment:
Is needed to return "null"? If the answer is: "Yes, it is", what do you think about not use "else" closure, instead you can do:

```suggestion
    if (res.status === STATUS_OK) {
      return res.data.data;
    }
    
    return null;
```

Reply:
I was thinking about this, I think the else branch is a good opportunity to log or send a notification, in the sense that instead of "hide" the problem, surface it somehow.

Maybe in the end the ideal solution is to handle all or most of the cases when the request fails.

Can you make a new issue to improve these calls, please @Katsu08?